### PR TITLE
feat: connected TS

### PR DESCRIPTION
**Pull request by Aleksey28**
1. Installed TS
2. Renamed files extension
3. Configurated settings files
4. Adapted cod with TS

### CODE REVIEW

**Review by AndreyBelym — COMMENTED** [review]
Awesome work! But unfortunately, ESLint doesn't suport TypeScript out-of-the box, you have to install a third-party plugin: typescript-eslint. Try to make a linting error on purpose and check it, I'm 99& sure that the current project configuration cannot catch it.

And after installing typescript-eslint, let's enable some additional rules and make your code compliant to them 😄 :
https://github.com/DevExpress/testcafe/blob/aac565a8966a0cc7b7453beb0a61158ade082827/.eslintrc#L2
https://github.com/DevExpress/testcafe/blob/aac565a8966a0cc7b7453beb0a61158ade082827/.eslintrc#L12
https://github.com/DevExpress/testcafe/blob/aac565a8966a0cc7b7453beb0a61158ade082827/.eslintrc#L95

**Comment by Aleksey28** [comment]
FPR ESlint for TS was updated.

File: test/config-test.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import defaultConfig, { debugConfig, default as config, host, port } from '../src/config.js';
+import defaultConfig, {debugConfig, default as config, host, port} from '../src/config.js';
```
Comment (AndreyBelym) [inline test/config-test.ts]: Hmm, why did spaces inside braces disappear? Does ESLint really cover this file?

BTW, ww should add spaces between braces ({ }) and the content that they surround.

**Comment by Aleksey28** [comment]
FPR

**Review by AndreyBelym — APPROVED** [review]

**Pull request merged**